pegasus/sites.v3/hourofcode.com/views/mobilecheck.js.haml

```diff
@@ -0,0 +1,13 @@
+:javascript
```

Comment:
Obviously not ideal to dupe but figuring out a way to share partials between code.org & hourofcode.com is a bit out of scope for this change. :/